flags/gcov.cmake

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2013, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_FLAGS_GCOV_CMAKE)
+  return()
+else()
+  set(POLLY_FLAGS_GCOV_CMAKE 1)
+endif()
+
+include(polly_add_cache_flag)
+
+polly_add_cache_flag(CMAKE_CXX_FLAGS "-fprofile-arcs -ftest-coverage -O0")
```

Comment:
You should add each flags one by one since this function will check that flag specified by an argument already exists.

For example:

```cmake
polly_add_cache_flag(CMAKE_CXX_FLAGS "-fprofile-arcs -ftest-coverage -O0")
polly_add_cache_flag(CMAKE_CXX_FLAGS "-ftest-coverage -fprofile-arcs -O0")
```
result:
```
CMAKE_CXX_FLAGS: -ftest-coverage -fprofile-arcs -O0 -fprofile-arcs -ftest-coverage -O0
```

separate version:
```cmake
polly_add_cache_flag(CMAKE_CXX_FLAGS "-fprofile-arcs")
polly_add_cache_flag(CMAKE_CXX_FLAGS "-ftest-coverage")
polly_add_cache_flag(CMAKE_CXX_FLAGS "-O0")

polly_add_cache_flag(CMAKE_CXX_FLAGS "-ftest-coverage")
polly_add_cache_flag(CMAKE_CXX_FLAGS "-fprofile-arcs")
polly_add_cache_flag(CMAKE_CXX_FLAGS "-O0")
```

result:
```
CMAKE_CXX_FLAGS: -O0 -ftest-coverage -fprofile-arcs
```